### PR TITLE
Compose 1.8.0-alpha03

### PR DESCRIPTION
**Pull request by MatkovIvan**
# DO NOT SQUASH! SHOULD BE MERGED MANUALLY

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -104,8 +104,8 @@ artifactRedirecting.publication=true
 artifactRedirecting.androidx.compose.version=1.8.0-alpha02
```
Comment (eymar) [inline gradle.properties]:
```suggestion
artifactRedirecting.androidx.compose.version=1.8.0-alpha03
```

here too, right?

File: compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/platform/Actuals.jsNative.kt [hunk 2]
```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+@file:Suppress("ACTUAL_ANNOTATIONS_NOT_MATCH_EXPECT")
```
Comment (eymar) [inline compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/platform/Actuals.jsNative.kt]: IDE says it's a redundant suppression. 

The expect is:
```
internal expect fun Any.nativeClass(): Any
```

Looks like the actual matches the expect. Why was the suppress needed?

**Review by eymar — APPROVED** [review]
LGTM.
1 desktop test is failing. But afaik we can merge the integration and add a ticket to fix the test later.

Note:
- added 1 commit to fix collections web tests compilation

File: compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/platform/Actuals.jsNative.kt (hunk 2 above)
Comment (MatkovIvan) [reply compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/platform/Actuals.jsNative.kt]: It's from https://github.com/JetBrains/compose-multiplatform-core/commit/6afa1f98324a5dbc909c84f422c47f2e94c0601f based on git blame. It looks like git considered this as part of file rename at some point

**Pull request merged**